making updates for the policy generator doc (#3496)
open-cluster-management/rhacm-docs: governance/policy_generator.adoc

          severity: low
----

> All policies where the input is based on {ocp-short} and are generated by the policy generator are fully supported. View the following examples of YAML input that is supported by {ocp-short}:

Reviewer: ```suggestion
All policies where the input is from the {ocp-short} documentation and are generated by the policy generator are fully supported. View the following examples of YAML input that is supported in the {ocp-short} documentation:
```
